src/lib.rs

```diff
@@ -36,5 +38,361 @@ pub fn y_py(_py: Python, m: &PyModule) -> PyResult<()> {
+            65, 10, 95, 95, 108, 105, 115, 116, 84, 121, 112, 101, 1, 119, 6, 110, 117, 109, 98,
+            101, 114, 40, 0, 156, 181, 228, 207, 14, 65, 5, 95, 95, 116, 97, 103, 1, 119, 2, 111,
+            108, 40, 0, 156, 181, 228, 207, 14, 65, 7, 95, 95, 115, 116, 97, 114, 116, 1, 125, 1,
+            7, 0, 156, 181, 228, 207, 14, 65, 6, 40, 0, 156, 181, 228, 207, 14, 73, 6, 95, 95, 116,
+            121, 112, 101, 1, 119, 8, 108, 105, 115, 116, 105, 116, 101, 109, 40, 0, 156, 181, 228,
+            207, 14, 73, 8, 95, 95, 102, 111, 114, 109, 97, 116, 1, 125, 0, 40, 0, 156, 181, 228,
+            207, 14, 73, 8, 95, 95, 105, 110, 100, 101, 110, 116, 1, 125, 0, 33, 0, 156, 181, 228,
+            207, 14, 73, 5, 95, 95, 100, 105, 114, 1, 40, 0, 156, 181, 228, 207, 14, 73, 7, 95, 95,
+            118, 97, 108, 117, 101, 1, 125, 1, 168, 156, 181, 228, 207, 14, 77, 1, 119, 3, 108,
+            116, 114, 7, 0, 156, 181, 228, 207, 14, 73, 1, 40, 0, 156, 181, 228, 207, 14, 80, 6,
+            95, 95, 116, 121, 112, 101, 1, 119, 4, 116, 101, 120, 116, 40, 0, 156, 181, 228, 207,
+            14, 80, 8, 95, 95, 102, 111, 114, 109, 97, 116, 1, 125, 0, 40, 0, 156, 181, 228, 207,
+            14, 80, 7, 95, 95, 115, 116, 121, 108, 101, 1, 119, 0, 40, 0, 156, 181, 228, 207, 14,
+            80, 6, 95, 95, 109, 111, 100, 101, 1, 125, 0, 40, 0, 156, 181, 228, 207, 14, 80, 8, 95,
+            95, 100, 101, 116, 97, 105, 108, 1, 125, 0, 132, 156, 181, 228, 207, 14, 80, 1, 100,
+            129, 156, 181, 228, 207, 14, 73, 1, 0, 5, 129, 156, 181, 228, 207, 14, 59, 1, 0, 5,
+            168, 156, 181, 228, 207, 14, 63, 1, 126, 135, 156, 181, 228, 207, 14, 13, 6, 40, 0,
+            156, 181, 228, 207, 14, 100, 6, 95, 95, 116, 121, 112, 101, 1, 119, 9, 112, 97, 114,
+            97, 103, 114, 97, 112, 104, 40, 0, 156, 181, 228, 207, 14, 100, 8, 95, 95, 102, 111,
+            114, 109, 97, 116, 1, 125, 0, 40, 0, 156, 181, 228, 207, 14, 100, 8, 95, 95, 105, 110,
+            100, 101, 110, 116, 1, 125, 0, 33, 0, 156, 181, 228, 207, 14, 100, 5, 95, 95, 100, 105,
+            114, 1, 168, 156, 181, 228, 207, 14, 104, 1, 119, 3, 108, 116, 114, 7, 0, 156, 181,
+            228, 207, 14, 100, 1, 40, 0, 156, 181, 228, 207, 14, 106, 6, 95, 95, 116, 121, 112,
+            101, 1, 119, 4, 116, 101, 120, 116, 40, 0, 156, 181, 228, 207, 14, 106, 8, 95, 95, 102,
+            111, 114, 109, 97, 116, 1, 125, 0, 40, 0, 156, 181, 228, 207, 14, 106, 7, 95, 95, 115,
+            116, 121, 108, 101, 1, 119, 0, 40, 0, 156, 181, 228, 207, 14, 106, 6, 95, 95, 109, 111,
+            100, 101, 1, 125, 0, 40, 0, 156, 181, 228, 207, 14, 106, 8, 95, 95, 100, 101, 116, 97,
+            105, 108, 1, 125, 0, 132, 156, 181, 228, 207, 14, 106, 1, 101, 1, 156, 181, 228, 207,
+            14, 9, 17, 1, 25, 1, 27, 7, 42, 2, 53, 6, 63, 1, 77, 1, 87, 12, 104, 1,
+        ];
+        let expected_json: serde_json::Value = json!({
```

Comment:
Uppercase constant?

Reply:
Not sure constant will work in this case.